cql3/statements/create_index_statement.cc

```diff
@@ -70,7 +70,8 @@ create_index_statement::validate(query_processor& qp, const service::client_stat
         throw exceptions::invalid_request_exception("Only CUSTOM indexes can be created without specifying a target column");
     }
 
-    _properties->validate();
+    gms::feature_service fs = gms::feature_service(gms::feature_config_from_db_config(qp.db().get_config()));
```

Comment:
This is absolutely not the right way to use `feature_service`. There should be only one instance of the feature service on each shard, and this instance is initialized in `main.cc`. You should not create your own instances, except during initialization code.

I suggest doing the validation in `validate_while_executing` instead - there, you can get access to the feature service via the `data_dictionary::database` object.